rpc/rpcserver/server.go

```diff
@@ -988,6 +988,8 @@ func marshalTransactionOutputs(v []wallet.TransactionSummaryOutput) []*pb.Transa
 			Index:    output.Index,
 			Account:  output.Account,
 			Internal: output.Internal,
+			Amount:   int64(output.Amount),
+			Address:  output.Address.String(),
```

Comment:
Needs to check for a nil address since not all scripts may be convertable to an address